client/src/lib/solana/model/TransactionAccountMeta.ts

```diff
@@ -1,5 +1,6 @@
 import { AccountMeta, PublicKey } from '@solana/web3.js';
 import { add_struct_to_schema, Assignable } from '../solanaBorsh';
+import { randomInt } from 'crypto';
```

Comment:
is this going to be an issue when running on the browser?

Reply:
The random function should probably not be here, it's for tests only. I'll move it to test utilites